csharp/Platform.Ranges/Range[T].cs

```diff
@@ -19,22 +19,25 @@ public struct Range<T> : IEquatable<Range<T>>
         private static readonly EqualityComparer<T> _equalityComparer = EqualityComparer<T>.Default;
 
         /// <summary>
-        /// <para>Returns minimum value of the range.</para>
-        /// <para>Возвращает минимальное значение диапазона.</para>
+        /// <para>Represents a minimum value of the range.</para>
+        /// <para>Представляет минимальное значение диапазона.</para>
         /// </summary>
         public readonly T Minimum;
 
         /// <summary>
-        /// <para>Returns maximum value of the range.</para>
-        /// <para>Возвращает максимальное значение диапазона.</para>
+        /// <para>Represents a maximum value of the range.</para>
+        /// <para>Представляет максимальное значение диапазона.</para>
         /// </summary>
         public readonly T Maximum;
 
         /// <summary>
-        /// <para>Initializes a new instance of the Range class.</para>
-        /// <para>Инициализирует новый экземпляр класса Range.</para>
+        /// <para>Initializes a new instance of the <see cref="Range{T}"/> structure.</para>
```

Comment:
We have two different constructors, so we have to add `using ...` template at the end for both constructors.